modules/installation-vsphere-zone-enablement.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+//
+//* installing-vsphere-installer-provisioned-customizations.adoc
+//* installing-vsphere-installer-provisioned-network-customizations.adoc
+//* installing_vsphere/installing-vsphere.adoc
+//* installing-vsphere-network-customizations.adoc
+//* installing-restricted-networks-installer-provisioned-vsphere
+//* installing-restricted-networks-vsphere.adoc
+
+:_content-type: CONCEPT
+[id="installation-vsphere-zone-enablement_{context}"]
+= VMware vSphere zone enablement
```

Comment:
```suggestion
= VMware vSphere regions and zones
```
Suggestion for a slightly more descriptive heading

Reply:
Good suggestion. Implemented throughout.